docs/gates.md

```diff
@@ -20,87 +20,109 @@ nav_order: 3
 ## Introduction
 
 
-Logic gates are the basic building blocks of any digital system. 
-It is an electronic circuit having one or more than one input and only one output. The relationship between the input and the output is based on a certain logic. Based on this, logic gates are named as AND gate, OR gate, NOT gate etc.
+It is a device which performs logic operations on one or more logical inputs, and produces a single logical output.
```

Comment:
Logic Gates are devices which perform logical operations on one or more inputs and produces a single output.

Reply:
Done